DEVGUIDE.md

```diff
@@ -134,6 +159,11 @@ To use your custom build of `Fsc`, add the `DotnetFscCompilerPath` property to y
 </PropertyGroup>
 ```
 
+## Updating parser and lexer
+
+Parser and lexer outputs are cached in the repository and are regenerated automatically every time fslexyacc input files are changed (fsl and fsy files), however, they will not be regenerated if any dependent compiler libraries are changed.
+If you are making changes to the latter, and do not see your changes in the parser/lexer, you will need to manually regenerate the outputs. This can be done by removing `src\.autogenerated\fslexyacc` directory, and running build again.
```

Comment:
Is there a command (something `dotnet msbuild` maybe) we can run instead to regenerate this?

Reply:
Not oob, maybe FsLex/FsYacc tasks can be modified to check some flag.